code/game/objects/items/devices/radio/electropack.dm

```diff
@@ -222,3 +223,82 @@ Code:
 			onclose(user, "radio")
 			return
 
+/obj/item/device/electropack/shockcollar/proc/remove_id()
+	if (id)
+		if (ismob(loc))
+			var/mob/M = loc
+			M.put_in_hands(id)
+			to_chat(usr, "<span class='notice'>You remove the ID from the [name].</span>")
+		else
+			id.loc = get_turf(src)
+		id = null
+
+/obj/item/device/electropack/shockcollar/verb/verb_remove_id()
```

Comment:
Upstream has taught me that verbs are horrid but this is passable I guess.

Reply:
It worked in testing. I could prolly nix it though. Not like it's essential. Will note that PDA uses the same thing.